pkg/giantnetes/utils.go

```diff
@@ -136,7 +136,7 @@ func (u *Utils) checkClusterVersionSupport(cluster clusterWithProvider) (bool, e
 func (u *Utils) getEtcdTLSCfg(clusterID string) (*TLSClientConfig, error) {
 	k8sClient := u.K8sClient.K8sClient()
 	getOpts := metav1.GetOptions{}
-	secret, err := k8sClient.CoreV1().Secrets(secretNamespace).Get(fmt.Sprintf("%s-etcd", clusterID), getOpts)
+	secret, err := k8sClient.CoreV1().Secrets(secretNamespace).Get(fmt.Sprintf("%s-calico-etcd-client", clusterID), getOpts)
```

Comment:
but this can be dangerous, as we plan to drop calico etcd client cert in the future since calico no longer is used as CNI for AWS

We might use this now, but we need some plan for the future, maybe custom cert for this?

Reply:
just being aware of that helps :)